_common.sh

```diff
@@ -192,15 +192,18 @@ function prepare_tomcat {
 
 	configure_tomcat
 
-	warm_up_tomcat
+	if [[ ! " ${@} " =~ " --no_warm_up " ]]
+	then
+		warm_up_tomcat
+	fi
 
 	rm -fr "${TEMP_DIR}"/liferay/logs/*
 	rm -fr "${TEMP_DIR}"/liferay/tomcat/logs/*
 }
 
-function remove_temp_dockerfile_target_platform {
-	sed -i 's/${TARGETARCH}'/$(get_current_arch)/ "${TEMP_DIR}"/Dockerfile
-	sed -i 's/--platform=${TARGETPLATFORM} //g' "${TEMP_DIR}"/Dockerfile
+function remove_temp_dockerfile_platform_variable {
+	sed -i'.bak' 's/${TARGETARCH}'/$(get_current_arch)/ "${TEMP_DIR}"/Dockerfile
+	sed -i'' 's/--platform=${TARGETPLATFORM} //g' "${TEMP_DIR}"/Dockerfile
```

Comment:
```suggestion
	sed -i'.bak' 's/--platform=${TARGETPLATFORM} //g' "${TEMP_DIR}"/Dockerfile
```
There is another one :)